# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/jit-test/tests/debug/Frame-01.js
# Commit: 73c33bf110d6
# Full Hash: 73c33bf110d6034266cbe615b9e018476a128d63
# Author: Logan Smyth <loganfsmyth@gmail.com>
# Date: 2019-12-09 09:50:39
# Regressor Bug: 1600204
# File Overlap Count: 2
# Description:
#   Bug 1600204 - Part 2: Remove Debugger.Frame.prototype.generator since it is already on script. r=jimb
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D55232
# ==============================================================================

diff -r d70ec1e79887 -r 73c33bf110d6 js/src/jit-test/tests/debug/Frame-01.js
--- a/js/src/jit-test/tests/debug/Frame-01.js	Fri Dec 06 02:29:18 2019 +0000
+++ b/js/src/jit-test/tests/debug/Frame-01.js	Fri Dec 06 02:57:20 2019 +0000
@@ -1,4 +1,4 @@
-// Test .type and .generator fields of topmost stack frame passed to onDebuggerStatement.
+// Test .type fields of topmost stack frame passed to onDebuggerStatement.
 
 var g = newGlobal({newCompartment: true});
 var dbg = Debugger(g);
@@ -6,7 +6,7 @@
 dbg.onDebuggerStatement = function (f) {
     assertEq(Object.getPrototypeOf(f), Debugger.Frame.prototype);
     assertEq(f.type, expected.type);
-    assertEq(f.generator, expected.generator);
+    assertEq(f.script.isGeneratorFunction, expected.generator);
     assertEq(f.constructing, expected.constructing);
     hits++;
 };